site/content/tutorial/02-reactivity/02-reactive-declarations/text.md

```diff
@@ -11,7 +11,7 @@ let count = 0;
 $: doubled = count * 2;
 ```
 
-> Don't worry if this looks a little alien. It's [valid](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label) (if unconventional) JavaScript, which Svelte interprets to mean 're-run this code whenever any of the referenced values change'. Once you get used to it, there's no going back.
+> Don't worry if this looks a little alien. It's [valid](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label) (though unconventional) JavaScript, which Svelte interprets to mean *re-run this code whenever any of the referenced values change*. Once you get used to it, there's no going back.
```

Comment:
The previous sentence was correct already, I don't think we need to change this.